compose/ui/ui/src/uikitInstrumentedTest/kotlin/android/compose/ui/keyboard/KeyboardInsetsTest.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.window
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.ime
+import androidx.compose.foundation.layout.windowInsetsPadding
+import androidx.compose.material.TextField
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.layout.boundsInRoot
+import androidx.compose.ui.layout.boundsInWindow
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.test.runUIKitInstrumentedTest
+import androidx.compose.ui.uikit.OnFocusBehavior
+import androidx.compose.ui.unit.DpRect
+import androidx.compose.ui.unit.height
+import androidx.compose.ui.unit.toDpRect
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlinx.coroutines.launch
+
+class KeyboardInsetsTest {
+    @Test
+    fun testBottomIMEInsets() = runUIKitInstrumentedTest {
+        var contentFrame: DpRect? = null
+
+        setContent {
+            Box(Modifier
+                .fillMaxSize()
+                .windowInsetsPadding(WindowInsets.ime)
+                .onGloballyPositioned { coordinates ->
+                    contentFrame = coordinates.boundsInWindow().toDpRect(density)
+                }
+            )
+            LaunchedEffect(Unit) {
+                launch {
+                    showKeyboard(animated = false)
+                }
+            }
+        }
+
+        waitForIdle()
```

Comment:
`setContent` should already `waitForIdle` before returning.

Reply:
Done